inc/class-wp-forms-api.php

```diff
@@ -925,6 +927,29 @@ static function process_element( $element, &$values, &$input ) {
 	}
 
 	/**
+	 * Get WP editor templates for dynamically-inserted editors in #multiple fields
+	 *
+	 * @action wp_ajax_wp_form_get_mce_templates
+	 */
+	function ajax_get_mce_templates() {
+		$results = array();
+		$bodies = filter_input( INPUT_POST, 'content', FILTER_DEFAULT, FILTER_REQUIRE_ARRAY );
+
+		foreach ( $bodies as $id => $body ) {
+			ob_start();
+			$lines = preg_split( '/$/m', $body );
```

Comment:
Yeah but you don't actually need to break on that, you just need to count them - isn't it always going to be CRLF or just LF? Do some systems actually just do CR? In any case, you could switch it to `preg_match_all()` since the return value is number of matches (may need to add 1 tho)